org.cvfosammmm.Setzer.json

```diff
@@ -17,7 +17,7 @@
         "--env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux",
         "--env=TEXMFCACHE=$XDG_CACHE_HOME",
         "--env=LC_ALL=C",
-        "--env=TEXINPUTS=.:~/texmf/:",
+        "--env=TEXINPUTS=.:~/texmf/:$HOME/.texmf:$XDG_DATA_HOME/texmf",
```

Comment:
Did you mean to add `~/.local/share/texmf`?

Because inside the sandbox, `$XDG_DATA_HOME` is `~/.var/app/org.cvfosammmm.Setzer/data` and that doesn't seem like a very standard place to put your texmf stuff. :stuck_out_tongue:

Reply:
Yeah, I  guess so. I'll fix it.

Reply:
I'm not sure this will work. `$HOME` will not be substituted. Nor will $XDG_DATA_HOME.